student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,6 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING(my_username(),malloc_username_from_conf_file());
```

Comment:
`malloc_username_from_conf_file()`, as the name suggests this function malloc's data. This needs to be freed in the function implementation before the function exits.